docs/Information_nodes.rst

```diff
@@ -62,6 +66,10 @@ commands::
 
     --gres=gpu:a100:<number> --reservation=DGXA100
 
+.. note::
+   If you use these nodes, you are asked to please acknowledge NVIDIA's gift in
+   in any academic work that has benefited from it.
```

Comment:
It's not true for `cn-d[001-002]`, only for `cn-d[003-004]`. Let's keep this for another PR once the team will have set up the program.